docs/manual/manual_advanced_arming.rst

```diff
@@ -0,0 +1,756 @@
+then there is no arming throttle region and throttles cannot cause an arming transition. Otherwise, arming throttles can be used as previously described.
+
+.. figure:: ../_static/manual_images/arming/arming_arm_on_throttle_parameter.png
+    :align: center
+    :width: 60%
+    :alt: Arm on Throttle Parameter
+
+    Arm on Throttle Parameter in IQ Control Center
+
+.. _arming_consecutive_throttles:
+
+Consecutive Arming Throttles
+------------------------------
+**Users can also specify how many consecutive arming throttles the module needs to receive before the arming transition occurs.** This can be useful 
+if the module is in a high-noise situation where there is a danger of noise being interpreted as a throttle command, or as a general safety 
+feature to help prevent premature arming. 
+
+The number of consecutive arming throttles the module must receive in the arming throttle region can be configured in the General tab of the
+Control Center with the *Consecutive Arming Throttles to Arm* parameter, as shown below.
+
+.. figure:: ../_static/manual_images/arming/arming_consecutive_throttles.png
+    :align: center
+    :width: 60%
+    :alt: Consecutive Arming Throttles to Arm Parameter
+
+    Consecutive Arming Throttles to Arm Parameter in IQ Control Center
+
+**Note that the number of consecutive arming throttles received by the module resets whenever a throttle command outside of the arming throttle 
+region is received. The module must receive the specified number of consecutive arming throttles all in a row, with no throttle commands outside of 
+the arming throttle region coming in between them.** For example, if the module's *Consecutive Arming Throttles to Arm* parameter was set to 10, and 
+it received 9 throttle commands in the arming throttle region, 1 outside of the region, and finally 1 throttle command in the region, 
+the module would not arm. At the end of that sequence, the module has only received 1 consecutive arming throttle, because the throttle command 
+from outside of the arming region reset the count.
+
+**It is also important to note that the consecutive arming throttles must be from the same** :ref:`throttle source <throttle_sources>`. A change in the source of 
+arming throttles will cause the count to reset. For example, if 5 arming throttles are sent over DroneCAN, and then 1 arming throttle 
+is received over hobby PWM, the count of consecutive arming throttles will be 1. The change of source for the arming throttles caused a reset of the count. 
+
+**To summarize, the module must receive a user-configurable number of consecutive throttle commands all in the arming throttle region and 
+all from the same source to trigger an arming transition.**
+
+Arming Example
+----------------
+An example of a possible arming setup and procedure is outlined below to clarify how arming with throttle commands works.

```

Comment:
I don't think you need to say "to clarify how arming with throttle commands works" that's sort of implied by the fact that it's an example